object/object.go

```diff
@@ -62,6 +62,41 @@ func (o *Object) ToV2() *object.Object {
 	return (*object.Object)(o)
 }
 
+// CopyTo writes deep copy of the [Object] to dst.
+func (o Object) CopyTo(dst *Object) {
+	if dst == nil {
+		return
+	}
+
+	id := (*object.Object)(&o).GetObjectID()
```

Comment:
`id`, `sig` and `header` have reference fields, why don't we deep copy them?

Reply:
Good question, I will update

Reply:
Updated, Now we have a monster inside SDK.

I tend to think we should implement this `CopyTo` inside api-go lib, because SDK in many cases uses types from it. Not for all types, but for most frequently used in SDK. For instance, take from this PR

Reply:
https://github.com/nspcc-dev/neofs-api-go/issues/432